backend/src/test/kotlin/blue/mild/covid/vaxx/routes/ServiceRoutesTest.kt

```diff
@@ -0,0 +1,53 @@
+package blue.mild.covid.vaxx.routes
+
+import blue.mild.covid.vaxx.dto.response.ApplicationInformationDtoOut
+import blue.mild.covid.vaxx.dto.response.SystemStatisticsDtoOut
+import blue.mild.covid.vaxx.utils.ServerTestBase
+import io.ktor.http.HttpMethod
+import io.ktor.http.HttpStatusCode
+import io.ktor.server.testing.handleRequest
+import org.junit.jupiter.api.Test
+import org.kodein.di.DI
+import org.kodein.di.bind
+import org.kodein.di.singleton
+import kotlin.test.assertEquals
+
+
+class ServiceRoutesTest : ServerTestBase() {
+
+    private val versionDto = ApplicationInformationDtoOut("test")
+
+    // an example of overriding the DI container
+    override fun overrideDIContainer() = DI {
+        bind<ApplicationInformationDtoOut>() with singleton { versionDto }
+    }
+
+    @Test
+    fun `test status responds ok`() = withTestApplication {
```

Comment:
No, ja bych psal `should ...`, tj. `should return ok response` - proste at je to veta.